### PR TITLE
Alert and avatar component

### PR DESCRIPTION
**Pull request by ApurvaSawant11**
Created Alert and Avatar components.

You can see the component UI here: https://dazzle-ui.netlify.app/

### CODE REVIEW

File: Components/Alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    justify-content: space-between;
+    align-items: center;
+    width: 60%;
+}
+
+.alert-link{
+    color: var(--white);
+}
+
+.fa-times{
```
Comment (animeshgarai09) [inline Components/Alert/alert.css:16]: I believe you are using font awesome, in that case, use `<i>` tag to target

**Review by animeshgarai09 — CHANGES_REQUESTED** [review]
Compress images from any online image compression tool, your site loading speed will improve

File: Components/Alert/alert.css (hunk 1 above)
Comment (ApurvaSawant11) [reply Components/Alert/alert.css:16]: If I use `<i>` tag then that will get applied to all other icons in the component library. I want to apply `cursor: pointer` to specific icons only

Comment (animeshgarai09) [reply Components/Alert/alert.css:16]: Okay. it's good to go then👍🏻

**Review by Maianki — APPROVED** [review]
Looks good to me you can merge it.

**Pull request merged**